.github/CODEOWNERS

```diff
@@ -1043,7 +1043,7 @@ spec/controllers/v0/rated_disabilities_controller_spec.rb @department-of-veteran
 spec/controllers/v0/rated_disabilities_discrepancies_controller_spec.rb @department-of-veterans-affairs/benefits-management-tools-be @department-of-veterans-affairs/va-api-engineers @department-of-veterans-affairs/backend-review-group
 spec/controllers/v0/sign_in_controller_spec.rb @department-of-veterans-affairs/vsp-identity
 spec/controllers/v0/terms_of_use_agreements_controller_spec.rb @department-of-veterans-affairs/vsp-identity
-spec/controllers/v0/upload_supporting_evidences_controller_spec.rb @department-of-veterans-affairs/Disability-Experience @department-of-veterans-affairs/va-api-engineers @department-of-veterans-affairs/backend-review-group
```

Comment:
@NB28VT – Did you intentionally remove this file and owner group? There's a failing check related to this change because now there are 2 owners without a corresponding file.

Reply:
+ 1, this is basically a syntax error @NB28VT , the `@` symbol here must be proceeded by a filepath

Reply:
Thanks dudes, looks like [I got this figured out](https://github.com/department-of-veterans-affairs/vets-api/actions/runs/7010987122/job/19072686458?pr=13835)